x-pack/metricbeat/module/aws/cloudwatch/metadata/rds/rds.go

```diff
@@ -0,0 +1,74 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
I would prefer it like `x-pack/metricbeat/module/aws/cloudwatch/rds/metadata.go` as in https://github.com/elastic/beats/pull/26117/files#diff-805a6af1b4f635a349720802c1c2faa856bfa4b8365b369d5ab44f93e156df63

Reply:
done :)